src/js/actions/shapes.js

```diff
@@ -292,17 +288,56 @@ define(function (require, exports) {
                 eventName,
                 options.coalesce);
 
-            var colorPromise = layerActionsUtil.playSimpleLayerActions(document, layers, strokeObj, true, options);
+            
+            if (!color && options.enabled) {
```

Comment:
I guess this is a long-standing issue, but the JSDoc should reflect `color`'s optional nature